examples/training/classification/basic_imagenet_training.py

```diff
@@ -206,6 +213,9 @@ def augment(img, label):
 
 
 callbacks = [
+    callbacks.ReduceLROnPlateau(
+        monitor="val_loss", factor=0.3, patience=5, min_lr=0.0001
+    ),
     callbacks.EarlyStopping(patience=30),
```

Comment:
lets increase patience now that we use callbacks.ReduceLROnPlateau()

Reply:
I've updated this to 40 (basically meaning two consecutive learning rate drops without improvement)